### ISSUE TITLE
warning: control-enable is 'no' in the config file

### ISSUE THREAD

**Issue by chauhuuphat** [MatthewVance/unbound-docker#71]
Hi Folks

I have a new record to "a-record.conf" and wanna apply it, I tried bellow command and get a issue that related control-enable
root@f8573ef4f534:/opt/unbound# unbound-control reload
[1627981880] unbound-control[40:0] warning: control-enable is 'no' in the config file.
[1627981880] unbound-control[40:0] error: connect: Connection refused for 127.0.0.1 port 8953

Do you have any idea to apply new record to this docker container Unbound ?

**Comment by MatthewVance** [comment]
Deploying a new instance of the container is the easiest method. If you
want the remote control feature, you can enable that by providing a custom
unbound.conf file.

**Comment by chauhuuphat** [comment]
Hi Matthew
I changed "control-enable: yes" but the docker container Unbound cannot start, it got this error
```
[1627982965] unbound[1:0] error: Error for server-cert-file: /opt/unbound/etc/unbound/unbound_server.pem
[1627982965] unbound[1:0] error: Error in SSL_CTX use_certificate_chain_file crypto error:02001002:system library:fopen:No such file or directory
[1627982965] unbound[1:0] error: and additionally crypto error:20074002:BIO routines:file_ctrl:system lib
[1627982965] unbound[1:0] error: and additionally crypto error:140DC002:SSL routines:use_certificate_chain_file:system lib
```
BTW, do you have any way to apply new records with this container without restart container?

**Comment by MatthewVance** [comment]
Unbound control requires additional setup to work with Docker because by
default it only allows localhost. It also needs access to read key files.

You could modify the a-records file for the running container, but the
Unbound service needs to reload for it to apply the change. Having Unbound
control is the other method.

**Comment by chauhuuphat** [comment]
Thanks Mattew
I also tried to modify the a-records file (add a new record). Unfortunately, Unbound didn't query the new record that I added after start the container Unbound. When I try to restart container, Unbound can query the new record.
In this case, in the docker container Unbound, when we try to update a new record to file a-record.conf, what command we can do to force Unbound update such as "service unbound reload"?
